JiraReporter/RestApiRequests.cs

```diff
@@ -21,19 +21,43 @@ public static T ResolveRequest<T>(Policy policy, RestRequest request, bool isXml
             else
                 client.Authenticator = new HttpBasicAuthenticator(policy.Username, policy.Password);
 
-            var results = client.Execute(request).Content;
+            return client;
+        }
+
+        public static T ResolveRequest<T>(Policy policy, RestRequest request, bool isXml = false)
+        {
+            var client = ClientLogin(policy);
+            var response = client.Execute(request);
+
+            validateResponse(response);
 
             if (isXml)
-                return Deserialization.XmlDeserialize<T>(results);
+                return Deserialization.XmlDeserialize<T>(response.Content);
             else
-                return Deserialization.JsonDeserialize<T>(results);
+                return Deserialization.JsonDeserialize<T>(response.Content);
         }
 
-        public static JiraReporter.JiraModels.Project GetProject(Policy policy)
+        public static T ResolveJiraRequest<T>(Policy policy, RestRequest request) where T : new()
+        {
+            var client = ClientLogin(policy);
+            var response = client.Execute<T>(request);
+
+            validateResponse(response);
+
+            return response.Data;
+        }
```

Comment:
@vladichimescu 

Nu imi e clar care e diferenta dintre metoda ResolveJiraRequest si ResolveRequest. 
Daca RestClient suporta direct Execute<T>, care include si deserializarea, atunci de ce o mai facem noi in ResolveRequest?